plugins/ufm_log_analyzer_plugin/README.md

```diff
@@ -75,4 +75,17 @@ This logic will show links that:
 
 ![Tool flow](img/loganalzer.png)
 
+## Testing
 
+There is a folder named `unit_tests`, this folder contains some unit tests, to run the tests follow these steps:
+
+1. Make sure you have `pytest` installed. If not, you can install it using `pip`:
+
+   ```bash
+   pip install pytest
```

Comment:
It should be clear that you use the venv created for the tool, right? otherwise tests might fail due to missing dependencies

Reply:
added